src/structured-data/handler.js

```diff
@@ -193,11 +193,18 @@ export async function opportunityAndSuggestions(auditUrl, auditData, context) {
   }
 
   // Convert suggestions to errors
+  const errorIdMap = {};
```

Comment:
You're using a global map across all URLs. So if two pages have the same issue, the issue on the second page will get the number suffix. What we want is the suffix for two or more identical issues on the same page :)

Page is stored in `issue.pageUrl`.